.github/workflows/system_tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: cargo install flutter_rust_bridge_codegen --version 1.62.1
       - name: Start containers
         run: |
-          docker-compose up bitcoin electrs -d
+          docker-compose up
```

Comment:
The services have to come after the options - `docker compose up [OPTIONS] [SERVICE...]`

Also I guess we need chopsticks for the system tests. So `docker-compose up -d chopsticks` should be fine. Note, we can skip the `bitcoin` and `electrs` service as these are dependencies of `chopsticks` and will get started anways.